src/munge.spec.ts

```diff
@@ -31,6 +37,29 @@ const validateOfferCodecs = (offer: Sdp): boolean => {
   return true;
 };
 
+/**
+ * Check that the sdp offer contains rtcpFbValue or not.
+ *
+ * @param offer - The {@link Sdp} or {@link AvMediaDescription} to validate.
+ * @param rtcpFbValue - The rtcp-fb value to check.
+ * @returns True if the offer contains rtcp-fb value.
+ */
+const checkOfferContainsRtcpFeedbacks = (
+  offer: Sdp | AvMediaDescription,
+  rtcpFbValue: string
+): boolean => {
+  let bContains = false;
+  const mediaDescriptions = offer instanceof Sdp ? offer.avMedia : [offer];
+  mediaDescriptions.forEach((av: AvMediaDescription) => {
```

Comment:
It doesn't matter here since it's a spec file so you don't need to change it, but a minor optimization would be to use `every` instead of `forEach`, since `every` would break out of the loop as soon as it finds a value that fails the condition.

Reply:
updated to use some() here.

Reply:
I wanted to do this either but I'll get an eslint warning then saying "Array.prototype.some() expects a return value from arrow function.eslint[array-callback-return](https://eslint.org/docs/rules/array-callback-return)"
as discussed, will just keep the return here.